bionlp/evaluate/evaluation.py

```diff
@@ -65,23 +63,23 @@ def get_Approx_Metrics(y_true, y_pred, verbose=True, preMsg='', flat_list=False)
     return avg_f1
 
 
-def get_ConfusionMatrix(true, predicted, final_eval=False):
+def get_ConfusionMatrix(true, predicted, final_eval=False, final_eval_out_file=''):
     # Confusion Matrix is only valid for partial evaluation.
     true_chain = list(itertools.chain.from_iterable(true))
     predicted_chain = list(itertools.chain.from_iterable(predicted))
     msg = "Confusion Matrix of combined folds (partial evaluation)\n{0}".format(
         ConfusionMatrix(true_chain, predicted_chain))
     print(msg)
-    if final_eval and final_eval_out_file:
-        append_message_to_final_eval_file(msg)
+    if final_eval and final_eval_out_file is not "":
```

Comment:
I'm not sure if I'm right, but it seems like `final_eval` and `final_eval_out_file` redundant. Isn't it enough to only check whether `final_eval_out_file is ""`?